extensions-contrib/tdigestsketch/pom.xml

```diff
@@ -29,7 +29,7 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>io.druid.extensions.contrib</groupId>
+  <groupId>org.druid.extensions.contrib</groupId>
```

Comment:
Thanks for the catch. This should be `org.apache.druid.extensions.contrib`, though.

Reply:
Would you please adjust it?

Reply:
I can't believe I missed this, thanks though.